pontoon/base/models.py

```diff
@@ -1082,6 +1082,7 @@ class Repository(models.Model):
         choices=TYPE_CHOICES
     )
     url = models.CharField("URL", max_length=2000)
+    branch = models.CharField("Branch", default='master', max_length=2000)
```

Comment:
This field should be made optional (`blank=True`) and the default parameter should be removed. We currently don't specify the branch, which means the default branch is used, which is not always called master (e.g. https://github.com/mozilla/activate.mozilla.community/).

This will probably require some changes to pontoon/sync/vcs/repositories.py.